scrapy/spidermiddlewares/__init__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+
+class BaseSpiderMiddleware(object):
+
+    def process_spider_input(self, response, spider):
+        # Called for each response that goes through the spider
+        # middleware and into the spider.
+
+        # Should return None or raise an exception.
```

Comment:
This needs to be re-implemented of course when creating a spider, but i'm a bit confused it is stated that None should be returned while the method here is passed. Isn't it more obvious to return None here